lib/k8sClient.js

```diff
@@ -89,7 +91,11 @@ module.exports = {
             path = `namespaces/${req.params.shipment}-${req.params.environment}/pods`,
             kube = k8s.api({
                 endpoint: apiHost,
-                version: 'v1'
+                version: '/api/v1',
```

Comment:
YUCK!

I know that it's because the `k8s` library changed, but that's terrible that they require a path for "version".